src/tests/basic/unit.ts

```diff
@@ -62,12 +62,35 @@ const debugGotoNextTurn = (
 };
 
 describe("basic", () => {
-  describe("unit", () => {
-    let $: GameState;
-    beforeEach(() => {
-      $ = initDummyGameState();
+  let $: GameState;
+  beforeEach(() => {
+    $ = initDummyGameState();
+  });
+  describe("card", () => {
+    it("can't play card not in hand", () => {
+      expect(() => performAction($, {
+        type: "PLAY_CARD",
+        cardID: "Nautical Dog",
+        boost: false,
+      })).to.throw("card not in hand");
+    });
+
+    it("played card is removed from hand", () => {
+      debugPlayCard($, "Nautical Dog");
+
+      const P = $.players[$.activePlayer];
+      expect(P.hand.pop() === "Nautical Dog").to.equal(false);
     });
```

Comment:
These two tests are good, but should be in a different file. This file is intended to hold all tests that deal with things unique to units (like attacking, patrolling, that sort of thing). Maybe we should have a file for tests that show whether the hand / discard / deck function as expected?

Reply:
basic/hand.ts :)

Reply:
Now on master as of my draw/discard PR being merged.

Reply:
I moved these into the `hand.ts` file along with adding a "can't play card without having enough money" test. I moved three methods out of the `unit.ts` file into the test helper one, because I was using the `debugPlayCard` method. If you think it doesn't make sense for those to live in testhelper I can move them back and stop using `debugPlayCard`.